Common/Source/Devices/devVolkslogger.cpp

```diff
@@ -123,7 +123,7 @@ BOOL VLDeclare(PDeviceDescriptor_t d, Declaration_t *decl, unsigned errBufferLen
   TCHAR2usascii(decl->CompetitionClass, vl.declaration.flightinfo.competitionclass, 13);
   vl.declaration.flightinfo.competitionclass[12]='\0'; // BUGFIX 100331
 
-  if (ValidWayPoint(HomeWaypoint)) {
+  if (ValidWayPoint(HomeWaypoint) && DeclTakeoffLanding) {
```

Comment:
are you sure ?

"vl.declaration.flightinfo.homepoint.name" is not part of turnpoint list.

Reply:
No I'm not sure but,  but what is this declaration about? What is it good for?
It is also skipped if no valid homewaypoint is defined.

Reply:
changed back